pages/mobile/base.py

```diff
@@ -78,6 +78,11 @@ def go_to_first_free_app_page(self):
         results_page = self.header.search(':free')
         return results_page.results()[0].click()
 
+    def go_to_app_that_is_guaranteed_to_contain_all_optional_fields(self):
+        self.selenium.get('%s/app/2' % self.base_url)
```

Comment:
It comes with the @krupa satisfaction guarantee! ;)

I will add a comment to the code.